GaelAlburo/final-project
# Add Static Services Page to Frontend

**Description:**  
This PR introduces a static version of the services page for the project's frontend.

**Changes:**  
- Implemented the initial static layout for the services page, including structure and styling.  
- Documented key functions and variables in the services page code to enhance clarity for future developers.  
- Added filtering functionality to view certain services.

**Impact:**  
- Provides the foundation for a services page, which can be expanded with dynamic content in future updates.  
- Improves code maintainability through detailed comments.  

**Next Steps:**  
- Enhance the services page with a connection to the services API to delete all the hard-coded information.

**Testing:**  
1. Run the frontend and verify that the services page renders correctly, including the hero image.  
2. Confirm that the filtering of services functions correctly
